sdk/keyvault/azure-keyvault-secrets-apis/azure/keyvault/secrets/aio/client.pyi

```diff
@@ -0,0 +1,22 @@
+# Stubs for azure.keyvault.secrets.aio.client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._shared import AsyncKeyVaultClientBase
+from azure.keyvault.secrets.models import DeletedSecret, Secret, SecretProperties
+from datetime import datetime
+from typing import Any, AsyncIterable, Optional
+
+class SecretClient(AsyncKeyVaultClientBase):
+    async def get_secret(self, name: str, version: Optional[str]=..., **kwargs: Any) -> Secret: ...
+    async def set_secret(self, name: str, value: str, content_type: Optional[str]=..., not_before: Optional[datetime]=..., expires: Optional[datetime]=..., **kwargs: Any) -> Secret: ...
```

Comment:
We should be consistent with the Keys package regarding what we do with the optional parameters here.
If we decide to keep them positional, the ordering of `expires` and `not_before` should be the same.
Alternatively, if we move them to kwargs, we should do the same thing here.

(same thing applies to `update_secret`, and also to the sync client)

Reply:
should be renamted expires_on and not_before and expires_on should both be moved to kwargs

Reply:
#7797